src/Microsoft.DotNet.Tools.Compiler.Fsc/project.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "dotnet-compile-fsc",
+    "version": "1.0.0-*",
+    "compilationOptions": {
+        "emitEntryPoint": true
+    },
+    "dependencies": {
+        "NETStandard.Library": "1.0.0-rc2-23616",
+        
+        "Microsoft.Net.Compilers.netcore": "1.2.0-beta-20151215-01",
+
+        "Microsoft.DotNet.Cli.Utils": {
+            "type": "build",
+            "version": "1.0.0-*"
+        },
+        "Microsoft.Extensions.CommandLineUtils.Sources": {
+            "type": "build",
+            "version": "1.0.0-*"
+        },
+        "Microsoft.DotNet.Compiler.Common": "1.0.0-*"
```

Comment:
Maybe something on http://myget.org/gallery/fsharp-daily would suffice.

Otherwise, I would consider this a blocker. F# needs to be delivered with dotnet-cli and it needs to run on CoreCLR.